lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/QuestionState.java

```diff
@@ -19,8 +19,8 @@ public class QuestionState {
     private List<List<QuestionAnswerPair>> allQaPairs = new ArrayList<>();
 
     @JsonIgnore
-    public boolean allQuestionBatchSizesMatch(int expectedBatchSize) {
-        return allQaPairs.stream().allMatch(x -> x.size() == expectedBatchSize);
+    public boolean isQuestionReceivedBatchCountEqualTo(int expectedSize) {
+        return allQaPairs.size() == expectedSize;
```

Comment:
Can you explain the change in logic here? Thanks

Reply:
https://github.com/govuk-one-login/ipv-cri-kbv-api/pull/352/commits/29d736339885963e3e4b8bb9b810272449dbbaec